comment.py

```diff
@@ -1,32 +1,34 @@
-import config
+from config import bot_config
 
 
 class Comment:
-    def __init__(self, title, body, byline="", attribution=""):
+    def __init__(self, title: str, body: str, byline: str = "", attribution: str = ""):
```

Comment:
How is the typing enforced/verified? Can this be included as a step in the CI process to make it meaningful?

Reply:
In Python they serve as type hints because it doesn't actually have types. However, you can statically verify that the types are correct wherever they're used. Seems like mypy is the standard (I haven't maintained a Python repo for a while) so you can figure out how to add that to travis. It's more of a linting process, though.